jenkins/rpm/v2/post_install_script.sh

```diff
@@ -4,7 +4,7 @@ BIN_DIR=${HA_INSTALL_DIR}/bin
 mkdir -p ${BIN_DIR} ${RES_AGENT}
 
 SITE_PACKAGES=`python3 -c 'import sysconfig; print(sysconfig.get_paths()["purelib"])'`
-HA_SETUP=${SITE_PACKAGES}/ha/setup/ha_setup.py
+HA_SETUP=${SITE_PACKAGES}/ha/k8s_setup/ha_setup.py
 CLI_EXEC=${SITE_PACKAGES}/ha/cli/cortxha.py
 DYNAMIC_RA=${SITE_PACKAGES}/ha/resource/dynamic_fid_service_ra.py
```

Comment:
We can remove most of these files now right ?

Reply:
Done.